tests/data_platform/preprocessing/test_preprocess_bluesky.py

```
@@ -56,3 +58,35 @@ def test_preprocess_delegates_to_runner(
+
+
+VALID_BLUESKY_POST_TEXT = (
+    "This is a valid English bluesky post for preprocessing tests without any "
+    "links and with enough words that length and language checks both pass here."
+)
+
+
+class TestPreprocessRecordsCanonicalText:
+    """Tests that Bluesky preprocess output includes canonical text."""
+
+    def test_preprocessed_rows_include_text(self, data_root: Path) -> None:
```

Ruff (0.16.3)

[warning] Unused method argument: `data_root`

(ARG002)